### PR TITLE
feat: PBI 4 Register Frontend

### LINKED ISSUE #10: Create register frontend

### PR DESCRIPTION
**Pull request by luckysusanto**
### Describe your changes
Implement Register Frontend
Changed Login Frontend to work with Register

### Backlog name
PBI-4-register-frontend

### Acceptance Criteria
User can sign up on the website

### Preview Page
register_frontend_0
register_frontend_1
register_frontend_2
register_frontend_3
register_frontend_4

### Checklist
- [X] I have performed a self-review of my code
- [X] Code successfully run on local
- [X] Feature / changes tested on local
- [X] No failing unit test
- [ ] Passed UAT Test
- [X] Sonarqube tested 
- [ ] Required any changes to environment variable

### CODE REVIEW

File: src/routes/register/index.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+import PasswordInput from '../../components/forms/passwordinput/index.js'
+import { Button, Text  } from '@chakra-ui/react';
+import {Box, Flex, Grid, GridItem } from '@chakra-ui/layout';
+import { useForm } from 'react-hook-form';
+import { route } from 'preact-router';
+import ROUTE from '../../config/api/route.js';
+import * as axios from "axios";
+
+function Register() {
+    console.log("Entering Register")
```
Comment (andrew-susanto) [inline src/routes/register/index.js]:
```suggestion
```

File: src/routes/register/index.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            password: res.password,
+            password2: res.password2
+        }
+
+        axios.post(`https://api-staging.monapi.xyz/register/api`, data)
+        .then(data => {
+            route(ROUTE.LOGIN + "?isRegistered=true")
+        }) 
+        .catch((error) => {
+            console.log("ERROR FROM VIEW: ", error)
```
Comment (andrew-susanto) [inline src/routes/register/index.js]:
```suggestion
```

**Review by andrew-susanto — APPROVED** [review]

File: src/routes/register/index.js (hunk 1 above)
Comment (luckysusanto) [reply src/routes/register/index.js]: Gotcha

File: src/routes/register/index.js (hunk 2 above)
Comment (luckysusanto) [reply src/routes/register/index.js]: Gotcha

File: src/routes/register/index.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+import PasswordInput from '../../components/forms/passwordinput/index.js'
+import { Button, Text  } from '@chakra-ui/react';
+import {Box, Flex, Grid, GridItem } from '@chakra-ui/layout';
+import { useForm } from 'react-hook-form';
+import { route } from 'preact-router';
+import ROUTE from '../../config/api/route.js';
+import * as axios from "axios";
+
+function Register() {
+
```
Comment (HugoIr) [inline src/routes/register/index.js:15]:
```suggestion
    if (!isAuthenticate()) {
```

Comment (HugoIr) [reply src/routes/register/index.js:15]: perlu ditambahin ini biar authenticated user gabisa ke register page

**Review by HugoIr — CHANGES_REQUESTED** [review]

File: src/routes/register/index.js (hunk 3 above)
Comment (luckysusanto) [reply src/routes/register/index.js:15]: sep

Comment (HugoIr) [reply src/routes/register/index.js:15]: ato ga bgini -andrew
if (isAuthenticate()) {
        route(ROUTE.DASHBOARD)
    }

**Review by HugoIr — APPROVED** [review]
aman

**Pull request merged**